src/main/kotlin/uk/gov/dwp/dataworks/egress/services/impl/DataServiceImpl.kt

```diff
@@ -58,6 +59,10 @@ class DataServiceImpl(
         if (specification.controlFilePrefix?.isNotBlank() == true) {
             controlFileService.egressControlFile(egressed, specification)
         }
+
+        if (specification.manifestFileName?.isNotBlank() == true) {
+            manifestFileService.egressManifestFile(specification)
```

Comment:
We call egressManifestFile, but don't use the Pair it returns. Is that right?

Reply:
Yes, we don't need further to use these values, the service fails if it couldn't process manifest file. So no further action to be taken by DataService, the returning pair  is following the convention used by ControlFile.

Reply:
The returning pair of manifestFileService is the following the convention of controlFileService. The service fails if manifest file is not processed successfully, so no further action is taken outside of the manifestFileService.

Reply:
Given we don't implement the controlFileService interface, do we need to follow the convention? I can't see where we use the returned Pair, unless im missing something?